src/scripts/main.js

```diff
@@ -22,7 +22,18 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key of Object.keys(data)) {
+    const list = document.createElement('ul');
+    const item = document.createElement('li');
+
+    item.innerText = key;
+    list.append(item);
+    element.append(list);
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(item, data[key]);
+    }
+  }
 }
 
-createTree(tree, food);
+createTree(tree, food);
```

Comment:
add empty line at the end of file

Reply:
https://matvii1.github.io/js_tree-from-object-DOM/